packages/gqless/src/Error/index.ts

```diff
@@ -1,6 +1,6 @@
 import type { GraphQLError } from 'graphql';
 
-export class gqlessError extends Error {
+export class GQlessError extends Error {
```

Comment:
The convention I was indirectly using was that Types start with uppercase, and values start with lower-case, and also changing this class is a breaking change, but if you really want to change it,  it should be also provided the previous class with the `@deprecated` comment that points to the same class

Reply:
We have no users yet, so this shouldn't be an issue.